dispatcher/conf/magento-proxy.conf

```diff
@@ -23,4 +23,6 @@ ProxyPassReverse "/magento/graphql" "${magento_host}/graphql"
 ProxyPass "/magento/rest" "${magento_host}/rest"
 ProxyPassReverse "/magento/rest" "${magento_host}/rest"
 ProxyPass "/magento/img" "${magento_host}/media/catalog/product"
-ProxyPassReverse "/magento/img" "${magento_host}/media/catalog/product"
\ No newline at end of file
+ProxyPassReverse "/magento/img" "${magento_host}/media/catalog/product"
```

Comment:
Can't we just have one generic mapping here only up till `catalog` like `ProxyPassReverse "/magento/img" "${magento_host}/media/catalog"` instead of adding multiple.

Reply:
Good idea, and move the `product` and `category` sub-folders to the components code.

Reply:
Right, so I vote definitely for that. Either via this or separate PR.

Reply:
It would be good to add it as a separate issue :) with  additional details and changes requested mentioned in it.